client/src/context/productState.js

```diff
@@ -26,12 +27,32 @@ const ProductState = props => {
         } catch (err) {}
     };
 
+    const loadAllProducts = async filterObj => {
+        if (localStorage.getItem("token")) {
+            axios.defaults.headers.common[
```

Comment:
this should be done once on a higher level so that you don't have do this again somewhere else in order to make an authenticated call